pkgs/servers/asterisk/default.nix

```diff
@@ -117,6 +122,7 @@ let
     inherit version sha256;
     externals = {
       "externals_cache/pjproject-2.12.tar.bz2" = pjproject_2_12;
+      "externals_cache/pjproject-2.12.1.tar.bz2" = pjproject_2_12_1;
```

Comment:
Both pjsip 2.12 and 2.12.1 have significant CVEs, any idea if asterisk upstream have addressed them?